staging/src/k8s.io/apiserver/pkg/storage/cacher/delegator.go

```diff
@@ -243,31 +242,33 @@ func shouldDelegateListOnNotReadyCache(opts storage.ListOptions) bool {
 // NOTICE: Keep in sync with shouldListFromStorage function in
 //
 //	staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/list_work_estimator.go
-func shouldDelegateList(opts storage.ListOptions) bool {
+func shouldDelegateList(opts storage.ListOptions) (shouldDeletage, consistentRead bool) {
```

Comment:
What I don't like here, is that we diverge with the other function in APF.
Can we converge them?